chain/starknet/src/adapter.rs

```diff
@@ -0,0 +1,32 @@
+use graph::blockchain::{EmptyNodeCapabilities, TriggerFilter as TriggerFilterTrait};
+
+use crate::{
+    data_source::{DataSource, DataSourceTemplate},
+    Chain,
+};
+
+// Currently `TriggerFilter` is useless, as it doesn't really filter anything from the block
+// stream, and the complete stream is always consumed, regardless of what the subgraph is indexing.
+// This is very bad for indexing performance and must be implemented for it to be considered
+// production-ready.
+// TODO: implement trigger filter for much better performance
```

Comment:
Probably not worth leaving these TODOs in here

Reply:
Removed.